entityDumper/channelDataDumper.go

```diff
@@ -201,7 +203,13 @@ func processChannel(db *sql.DB, writer *bufio.Writer, channelLabel string,
 	schemaMetadata map[string]schemareader.Table, options DumperOptions) {
 	whereFilter := fmt.Sprintf("label = '%s'", channelLabel)
 	tableData := dumper.DataCrawler(db, schemaMetadata, schemaMetadata["rhnchannel"], whereFilter, options.StartingDate)
-	log.Debug().Msg("finished table data crawler")
+
+	totalRows := 0
+	for _, value := range tableData.TableData {
+		totalRows = totalRows + len(value.KeyMap)
+	}
+
+	log.Debug().Msgf("finished table data crawler. Total database rows to export: %d", totalRows)
```

Comment:
Nitpick, but this should have been under `if log.Debug().Enabled() {` i.e.

```suggestion
	if log.Debug().Enabled() {
		totalRows := 0
		for _, value := range tableData.TableData {
			totalRows = totalRows + len(value.KeyMap)
		}
		log.Debug().Msgf("finished table data crawler. Total database rows to export: %d", totalRows)
	}
```